source/frontend/src/features/mapSideBar/property/MotiInventoryHeader.tsx

```diff
@@ -63,6 +73,17 @@ export const MotiInventoryHeader: React.FunctionComponent<IMotiInventoryHeaderPr
               </HeaderField>
             </Col>
           </Row>
+          {isExpired && (
+            <Row className="no-gutters">
+              <Col xs="8"></Col>
+              <Col className="d-flex justify-content-end pr-4">
+                <ExpiredWarning>
+                  <AiOutlineExclamationCircle size={16} />
+                  &nbsp; EXPIRED
```

Comment:
not a fan of the "&nbsp;". wouldn't it be better to use CSS to fix?

Reply:
This is how we display on other pages (ie Leases) but will give it a margin instead